src/pages/getting-started/app-guides/fork-your-app/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+weight: 110
+category: "getting-started"
+subcategory: "app-guides"
+title: "Forking Your App"
+tagline: "Fork your app to be able to edit an already published app."
+desc: "Discover how to fork your app to be able to edit an already published app."
+---
+
+# Introduction
+
+There are situations when your app is already published on our <Placeholder id="LIVECHAT_MARKETPLACE_URL"/> or <a href="https://www.helpdesk.com/marketplace/" target="_blank">HelpDesk Marketplace</a>, but you decide it needs a change or improvement. To be able to introduce changes to your published app, you need to fork it and then replace the original app with the forked version. 
+
+# Forking process
+
+To start the forking process of your published app, you can choose between two options:
+
+1. You need to go to the **Apps** tab on the left of your screen and click the burger menu button of the app you want to fork. Then, click the **Edit app** button to proceed with the forking process.
+2. You need to go to the **Apps** tab on the left of your screen and click the app you want to fork. Next, click the **Manage App** button, pick the **Fork app** option, and follow the process.
+
+<Warning>
+
+**Credentials after forking an app**
+
+The forked app will receive new authorization credentials. However, the original credentials (**Client Id** and **Client Secret**) will remain unchanged when the fork is published.
```

Comment:
I'd add info the "new authorization credentials" are temporary for the fork, and will be removed as soon as fork merges with the original app.